src/components/CopyLink.js

```diff
@@ -22,8 +24,11 @@ export function CopyLink(props) {
     // Turn viewTarget into a URL query string
     const viewTarget = props.getCurrentViewTarget();
 
-    const params = new URLSearchParams(viewTarget).toString();
+    console.log("copying");
+    //const params = new URLSearchParams(viewTarget).toString();
+    const params = qs.stringify(viewTarget, { encode: false });
```

Comment:
Is there a reason we need to turn off `encode` here? That ought to be explained in a comment.